docs/_user_guide/best_practices_optimization.md

```diff
@@ -1,306 +1,354 @@
-We have licensed and installed optimization software on the Yens that the GSB researchers can use including 
-<a href="https://www.gurobi.com" target="_blank">Gurobi</a>, <a href="https://www.artelys.com/solvers/knitro" target="_blank">Knitro</a> and <a href="https://ampl.com" target="_blank">AMPL</a>. 
-In this guide, we will show how to 
-use the licensed optimization software interactively on the Yens, in Jupyter notebooks and in Slurm. 
+# Optimization
```

Comment:
I think overall, we can make this more streamlined by using tabs and instead of having repeat information. For example, a tab for Gurobi and a tab for ampl/knitro. OR a tab for python and a tab for R.

Reply:
That would be great, but I'm unsure how to do that since the license information, examples, compatibility issues, etc., seem different for these packages. They are not exactly written in a parallel format. If I work on it, I need more guidance.